acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/transaction/PantheonWeb3j.java

```diff
@@ -73,6 +73,40 @@ public PantheonWeb3j(
         SignersBlockResponse.class);
   }
 
+  public Request<?, ProposeResponse> ibftPropose(final String address, final Boolean auth) {
+    return new Request<>(
+        "ibft_proposeValidatorVote",
+        Arrays.asList(address, auth.toString()),
+        web3jService,
+        ProposeResponse.class);
+  }
+
+  public Request<?, DiscardResponse> ibftDiscard(final String address) {
+    return new Request<>(
+        "ibft_discardValidatorVote", singletonList(address), web3jService, DiscardResponse.class);
+  }
+
+  public Request<?, ProposalsResponse> ibftProposals() {
+    return new Request<>(
+        "ibft_getPendingVotes", emptyList(), web3jService, ProposalsResponse.class);
+  }
+
+  public Request<?, SignersBlockResponse> ibftGetValidators(final String blockNumber) {
+    return new Request<>(
+        "ibft_getValidatorsByBlockNumber",
+        singletonList(blockNumber),
+        web3jService,
+        SignersBlockResponse.class);
+  }
+
+  public Request<?, SignersBlockResponse> ibftGetSignersAtHash(final Hash hash) {
```

Comment:
Doesn't look this is being used. But we I think we should have a test to verify this rpc is working.